conf/runner.yaml

```diff
@@ -8,12 +8,30 @@ runs:
         - addon.mozilla.org
         - addons.mozilla.org
         - blocklist.addons.mozilla.org
-        - builder.addons.mozilla.org
         - services.addons.mozilla.org
         - static.addons.mozilla.net
         - versioncheck-bg.addons.mozilla.org
         - versioncheck.addons.mozilla.org
         - compatibility-lookup.services.mozilla.com
+        - addons.cdn.mozilla.net
```

Comment:
Add discovery.addons.mozilla.org? since we monitor its -dev and -stage endpoints below.